content/concepts/introduction/host.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "libp2p Host"
+description: "A libp2p host is a program or process that runs on a peer and allows it to participate in a libp2p network."
+weight: 4
+---
+
+A libp2p host is a program or process that allows a peer to participate
+in the libp2p network and defines the peer's properties. In the context of
+libp2p, a host is a specific implementation of a node that has been configured
+to use the libp2p stack.
```

Comment:
```suggestion
in the libp2p network.
```

Reply:
Maybe also mention that there are different implementations, and not all implementations support all the features listed below.